src/ui/components/menu/Options.tsx

```diff
@@ -36,6 +37,7 @@ export default class Options extends React.Component<IProps, IState> {
 
     this.state = {
       game_install_path: '',
+      game_executable: '',
```

Comment:
Again, there should only need to be one option for this. If you want to put functions that get the game folder/exe name/metadata path inside of their own file in `src/utils` you are more than welcome to